<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
    <item row="0" column="0">
     <widget class="QGroupBox" name="grpMeasParams">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Measurement parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
       <item row="1" column="1">
        <widget class="QSpinBox" name="eBeginFrequency">
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>19999</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="eVolumeDBFS">
         <property name="suffix">
          <string> dBFS</string>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>0.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>-12.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="eLength">
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lkSampleRate">
         <property name="text">
          <string>Sample rate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lkBeginFrequency">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLabel" name="lvDuration"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lkEndFrequency">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="eEndFrequency">
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>22000</number>
         </property>
         <property name="value">
          <number>20000</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lkDuration">
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lkFrequencyRange">
         <property name="text">
          <string>Frequency range</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="eSampleRate"/>
       </item>
       <item row="7" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="bMeasure">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lkLength">
         <property name="text">
          <string>Length</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lkVolume">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCustomPlot" name="plotResponse" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCustomPlot" name="plotOutputSpectrogram" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCustomPlot" name="plotInputSpectrogram" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 86, 86);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="bToggleSpectrogram">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Toggle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>eBeginFrequency</tabstop>
  <tabstop>eEndFrequency</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
